tests/end_to_end/conftest.py

```diff
@@ -270,16 +237,16 @@ def fx_federation(request, pytestconfig):
         model_owner.modify_plan(
             new_rounds=num_rounds,
             num_collaborators=num_collaborators,
-            disable_client_auth=disable_client_auth,
-            disable_tls=disable_tls,
+            require_client_auth=require_client_auth,
+            use_tls=use_tls,
         )
     except Exception as e:
         log.error(f"Failed to modify the plan: {e}")
         raise e
 
     # For TLS enabled (default) scenario: when the workspace is certified, the collaborators are registered as well
     # For TLS disabled scenario: collaborators need to be registered explicitly
-    if args.disable_tls:
+    if not use_tls:
         log.info("Disabling TLS for communication")
```

Comment:
It's not part of your changes, but let's remove this logging, as we print all the param values in the beginning.

Reply:
We discussed on this. Keeping this log stmt intact.

Reply:
Lets keep the logs.